core/src/main/scala/sbtorgpolicies/github/GitHubOps.scala

```diff
@@ -25,6 +25,7 @@ import com.github.marklister.base64.Base64._
 import github4s.Github
 import github4s.GithubResponses._
 import github4s.free.domain._
+// import sbt.io.IO
```

Comment:
The following
```scala
import sbt.io._
import sbt.io.syntax._
```
and the equivalent are the just "notes" and would be the imports if we were using purely sbt 1.0.

https://github.com/dwijnand/sbt-compat allows us to keep the equivalent sbt 0.13 imports.

Reply:
👍 I think we could starting using those soon

Reply:
Because of how `sbt-compat`, it exposes the sbt 1.0 api through the sbt 0.13 import statements. It's either that or have scala version specific source files where the only difference is something like
```diff
47deg-suhas:main suhas$ diff scala-2.10/sbtorgpolicies/io/io.scala scala-2.12/sbtorgpolicies/io/io.scala
24c24,25
< import sbt.{file, File}
---
> import sbt.io._
> import sbt.io.syntax._
```

I think we'll need to keep this until we stop supporting sbt 0.13.